backend/kesaseteli/settings.py

```diff
@@ -26,8 +27,7 @@
     USE_X_FORWARDED_HOST=(bool, False),
     DATABASE_URL=(
         str,
-        "postgres://kesaseteli:kesaseteli"
-        "@localhost/kesaseteli",
+        "postgres://kesaseteli:kesaseteli" "@localhost/kesaseteli",
```

Comment:
There's no point in string splitting if its on a single line, so combine it to single string instead.
Also, maybe the default value could be something simpler such as `postgres:///kesaseteli` which works nicely with a locally running PostgreSQL.